src/Controller/QuestionController.php

```diff
@@ -14,29 +17,28 @@ class QuestionController extends AbstractController
      */
     public function homepage(Environment $twigEnvironment)
```

Comment:
```suggestion
    public function homepage(Environment $twigEnvironment): ResponseInterfface
```
https://www.php.net/manual/ru/migration71.new-features.php